builders/build/substrate-api/chopsticks.md

```diff
@@ -0,0 +1,167 @@
+    EligibleCount: 100
+```
+
+The settings that can be included in the config file are the same as the flags in the [`yarn start dev` command](#forking-moonbeam){target=_blank}, as well as these additional options:  
+
+|          Option          |                                           Description                                               |
+|:------------------------:|:---------------------------------------------------------------------------------------------------:|
+|         genesis          | The link to a parachain's raw genesis file to build the fork from, instead of an endpoint.          |
+|        timestamp         | Timestamp of the block to fork from.                                                                |
+
+## Forking Moonbeam {: #forking-moonbeam }
+
+The simplest way to fork Moonbeam is through the previously introduced configuration files:  
+
+=== "Moonbeam"
+    ```
+    yarn start dev --config=configs/moonbeam.yml
+    ```
+
+=== "Moonriver"
+    ```
+    yarn start dev --config=configs/moonriver.yml
+    ```
+
+=== "Moonbase Alpha"
+    ```
+    yarn start dev --config=configs/moonbase-alpha.yml
+    ```
+
+A configuration file is not necessary, however. There are additional commands and flags to configure the environment completely in the command line.  
+
+The `yarn start dev` command forks a chain, and includes following flags:  
+
+|           Flag           |                                           Description                                               |
+|:------------------------:|:---------------------------------------------------------------------------------------------------:|
+|         endpoint         | The endpoint of the parachain to fork.                                                              |
+|          block           | Use to specify at which block hash or number to replay the fork.                                    |
+|      wasm-override       | Path of the WASM to use as the parachain, instead of an endpoint.                                   |
+|            db            | Path to the name of the file that stores or will store the parachain's database.                    |
+|          config          | Path to the config file.                                                                            |
+|           port           | The port to expose an endpoint on.                                                                  |
+|      build-block-mode    | How blocks should be built in the fork: batch, manual, instant.                                     |
+|      import-storage      | A pre-defined JSON/YAML storage file path to override in the parachain's storage.                   |
+| allow-unresolved-imports | Whether to allow WASM unresolved imports when using a WASM to build the parachain.                  |
+|       import-storage     | Include to generate storage diff preview between blocks.                                            |
```

Comment:
the flag "import-storage" is in both line 100 and line 102. Should line 102 be "html" according to "Use option --html to generate storage diff preview (add --open to automatically open file)" in https://github.com/AcalaNetwork/chopsticks?

Reply:
thanks! fixed in https://github.com/PureStake/moonbeam-docs/pull/620